### ISSUE TITLE
Help ASAP - LG A2 can't be detected or connected to ColorControl 9.9.0.1

### ISSUE THREAD

**Issue by irohxxo** [Maassoft/ColorControl#388]
I tried turning on network settings on from (networks) settings refreshed but no device discovery and I tried manually putting ip address and mac in colorcontrol but it says "unable to connect to the device".
How can I fix it?

I'm trying to enable G-Sync VRR on lg A2 if anyone tried it tell me if its possible and if its as good as on better models with official ones installed.

**Comment by Sugaroverdose** [comment]
OLED A1/A2 series does not have neither G-Sync nor VRR, they basically locked at 60hz.

Can't really tell why it's not visible in network tab, it might have specific model index which isn't recognized by software, just add it's ip address by hands.

Anyway i would advice returning this A series thing if you wanted to use it as gaming monitor and get C series instead.

**Comment by irohxxo** [comment]
> OLED A1/A2 series does not have neither G-Sync nor VRR, they basically locked at 60hz.
> 
> Can't really tell why it's not visible in network tab, it might have specific model index which isn't recognized by software, just add it's ip address by hands.
> 
> Anyway i would advice returning this A series thing if you wanted to use it as gaming monitor and get C series instead.

So there is no way of adding VRR to this TV even with this software?
I typed ip address as well but it says "unable to connect to the device".

**Comment by Sugaroverdose** [comment]
> So there is no way of adding VRR to this TV even with this software?

From what i know, HDMI VRR support and G-Sync compatibility appeared on LG OLED series only with HDMI 2.1 ports, A1/A2 lacks HDMI 2.1(they are both HDMI 2.0) and it's pretty much pointless for them to have VRR with 20hz window(bottom line is 40hz, top line is 60hz)

**Comment by irohxxo** [comment]
> > So there is no way of adding VRR to this TV even with this software?
> 
> From what i know, HDMI VRR support and G-Sync compatibility appeared on LG OLED series only with HDMI 2.1 ports, A1/A2 lacks HDMI 2.1(they are both HDMI 2.0) and it's pretty much pointless for them to have VRR with 20hz window(bottom line is 40hz, top line is 60hz)

yes but, that 20hz is important because compared to V-Sync, vrr has less latency penalty. and most graphics card can handle 4k games up to 60fps. so that vrr is important.

**Comment by irohxxo** [comment]
so nobody knows what can be an issue with connecting with color control?

**Comment by irohxxo** [comment]
hello???
